File: InsuranceCompany.Core/Models/InsuranceRate.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;

namespace InsuranceCompany.Core;

public partial class InsuranceRate
{
    [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
    public Guid Id { get; set; }

    public string Title { get; set; }
    public short? CountPaymentsInYear { get; set; }
    public short? CountYears { get; set; }
    public bool? IsFamily { get; set; }
    public bool? IsOldman { get; set; }
    public decimal? BaseCoefficient { get; set; }
    public bool? IsPersonal { get; set; }
    public virtual ICollection<InsuranceRequest> InsuranceRequests { get; } = new List<InsuranceRequest>();
}

File: InsuranceCompany.Client/src/app/models/InsuranceRate.ts
import { InsuranceTypeSurvey } from "./InsuranceTypeSurvey";

export class InsuranceRate {
    id : string;
    title : string;
    unitPayment : number;
    countPaymentsInYear : number;
    countYears : number;
    isFamily :boolean;
    cost: number;
    baseCoefficient: number;
    insuranceTypeSurveys: InsuranceTypeSurvey[];

    constructor(id: string, title : string, unitPayment : number,
        countPaymentsInYear : number, countYears : number, cost: number, baseCoefficient: number) 
    {
        this.title = title;
        this.unitPayment = unitPayment;
        this.countPaymentsInYear = countPaymentsInYear;
        this.countYears = countYears;
        this.cost = cost;
        this.baseCoefficient = baseCoefficient;
        this.isFamily = false;
    }
}